web/src/components/DatasetSubimageSelector.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div>
+    <v-tooltip
+      v-if="!selectionMode"
+      bottom
+    >
+      <template v-slot:activator="{ on, attrs }">
+        <v-btn
+          icon
+          v-bind="attrs"
+          v-on="on"
+          @click="() => store.commit.setSelectionMode(true)"
+        >
+          <v-icon>mdi-select</v-icon>
+        </v-btn>
+      </template>
+      <span>Select a portion of the data</span>
+    </v-tooltip>
+    <div v-else>
+      <v-btn
+        v-if="!selection"
+        color="secondary"
+        @click="() => store.commit.setSelectionMode(false)"
+      >
+        Cancel
+      </v-btn>
+      <v-btn
+        v-else
+        color="primary"
+        @click="saveSubimageDataset"
+      >
+        Create Subimage Dataset {{ selectionText() }}
+      </v-btn>
```

Comment:
```suggestion
      <v-btn
        color="secondary"
        @click="() => store.commit.setSelectionMode(false)"
      >
        Cancel
      </v-btn>
      <v-btn
        v-if="selection"
        color="primary"
        @click="saveSubimageDataset"
      >
        Create Subimage Dataset {{ selectionText() }}
      </v-btn>
```
I think the cancel button should still be enabled if there's a selection. That way a user can cancel creating the subimage if they've picked the "wrong" region.